registry/mock.go

```diff
@@ -17,8 +17,8 @@ type mockRemote struct {
 	err  error
 }
 
-type ManifestFunc func(id image.Ref) (image.Info, error)
-type TagsFunc func(id image.Name) ([]string, error)
+type ManifestFunc func(ref string) (image.Info, error)
+type TagsFunc func() ([]string, error)
```

Comment:
Do either of these types actually need to exist? I would guess they can be inlined.

Reply:
Still seems to be the case but they've moved.

Reply:
Yes they could just be the argument types.